### ISSUE TITLE
Featured Posts

### ISSUE THREAD

**Issue by SriSatyaLokesh** [thiagorossener/jekflix-template#98]
It would be much better if we can add featured posts feature to this theme

**Comment by thiagorossener** [comment]
What do you mean?

**Comment by SriSatyaLokesh** [comment]
If we have tag `featured` in tags.

we will show them separately. (These are like most common or posts with more priority)

**Comment by SriSatyaLokesh** [comment]
![image](https://user-images.githubusercontent.com/43106076/84812636-50457400-b02c-11ea-9612-5fbbda6de838.png)

**Comment by thiagorossener** [comment]
Thanks for the idea @SriSatyaLokesh, I need to put more thought on this.

Initially, I thought you were talking about being able to select the featured post instead of showing the latest one, but looks like the idea is to show a list of featured posts.

At the moment, I don't see how to fit those posts into the home page layout unless they come first on the page, but that would make things confusing, once the first posts would be the "featured" ones and after that it'd show the rest of them in chronological order.

Ideas are welcome.

**Comment by SriSatyaLokesh** [comment]
Yes That's a easy fix, making first post as featured post and rest all in chronological order.
But that won't help us upto the extent. Maybe when we use `two column layout`, we can show featured post down to the `see also posts`
It would be more better if we show `posts with same category in see also section`.

**Comment by thiagorossener** [comment]
Yeah, that makes sense. I'll mark it as `enhancement`, so I can hit it on the next update round.

**Comment by algorithmor** [comment]
+1 for the idea @SriSatyaLokesh. I agree with @thiagorossener on the point that it would make sense to have this section at the top.  So, can we have a slider card layout that can have multiple featured posts. Similar to what is shown in this video.
 
https://www.youtube.com/watch?v=_Tfz1DAygrQ

The card's width can be the same as the hero element and we can have a toggle to either include Hero element or have featured posts.

**Comment by algorithmor** [comment]
But, I see a challenge with the above option. Keeping single card element and then aligning it to the full width of the screen can present layout mismatch issues. So, perhaps a more refined approach would be to have a carousel of cards. The card/box sizes can be the same as now, but arranged similar to what's seen in this video. 

https://www.youtube.com/watch?v=77JkijEaFmQ
